tests/Integration/Schema/Directives/CountDirectiveDBTest.php

```diff
@@ -420,4 +420,67 @@ public function testResolveRelationItemsAndCount(): void
             ],
         ]);
     }
+
+    public function testCountModelWithSingleColumn(): void
+    {
+        $this->schema = /** @lang GraphQL */ '
+        type Query {
+            tasks: Int @count(model: "Task", columns: ["difficulty"])
+        }
+        ';
+        factory(Task::class, 3)->create();
+
+        $this->graphQL(/** @lang GraphQL */ '
+        {
+            tasks
+        }
+        ')->assertExactJson([
+            'data' => [
+                'tasks' => 3,
+            ],
+        ]);
+    }
+
+    public function testCountModelWithMultipleColumns(): void
```

Comment:
Same issue as with `testCountModelWithSingleColumn`. The test should show that `columns` makes a meaningful difference in results.